pkg/deploy/graph/analysis/dc.go

```diff
@@ -35,6 +38,24 @@ dc:
 				// The image stream for the tag of interest does not exist.
 				// TODO: Suggest `oc create imagestream` once we have that.
 				if isNode, exists := doesImageStreamExist(g, uncastIstNode); !exists {
+
+					// We may not be able to see the image stream because it's in different project.
+					for _, t := range dcNode.DeploymentConfig.Spec.Triggers {
+						if t.Type == deployapi.DeploymentTriggerOnImageChange &&
+							t.ImageChangeParams.From.Namespace != dcNode.DeploymentConfig.Namespace {
+							markers = append(markers, osgraph.Marker{
+								Node:         uncastDcNode,
+								RelatedNodes: []graph.Node{uncastIstNode, isNode},
+
+								Severity:   osgraph.WarningSeverity,
```

Comment:
This should be info.  We have no reason to believe there's anything wrong here.  I'd rather see an attempt to do the lookup.  A 404 means its broken a 403 means "maybe".